src/user/api/views.py

```diff
@@ -0,0 +1,19 @@
+from multiprocessing import get_context
+from rest_framework import generics
+from .serializers import UserLoginSerializer,UserSerializer
+from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework.response import Response
+
+class UserLoginView(generics.GenericAPIView):
+    serializer_class=UserLoginSerializer
+
+    def post(self,request,*args,**kwargs):
+        serializer=UserLoginSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        user=serializer.validated_data
+        token=RefreshToken.for_user(user)
```

Comment:
septate this to access and refresh token like simple jwt example:
https://django-rest-framework-simplejwt.readthedocs.io/en/latest/creating_tokens_manually.html